src/main/java/com/ibm/cloud/cloudant/kafka/connect/connectors/ChangesSourceConnector.java

```diff
@@ -11,10 +11,14 @@
  * either express or implied. See the License for the specific language governing permissions
  * and limitations under the License.
  */
-package com.ibm.cloud.cloudant.kafka.connect;
+package com.ibm.cloud.cloudant.kafka.connect.connectors;
```

Comment:
Looking at this again it does seem a bit over-long especially as it needs entering into config.
Maybe we should drop one of hte levels here.

Reply:
As @tomblench suggested, I will put them on the level `com.ibm.cloud.cloudant.kafka`

Reply:
It is fixed in [8bf0292](https://github.com/IBM/cloudant-kafka-connector/pull/110/commits/8bf0292e64bf6eb92a3a7a33f9e97bc90a65e83d)